src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java

```diff
@@ -28,14 +28,19 @@
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
     @Test
-    public void parse_validArgs_returnsDeleteCommand() {
+    public void parse_validArgsName_returnsDeleteCommand() {
         Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
         Name name = personToDelete.getName();
         DeleteCommand deleteCommand = new DeleteCommand(personToDelete.getName());
 
         assertParseSuccess(parser, " n/" + name.fullName, deleteCommand);
     }
 
+    @Test
+    public void parse_validArgsIndex_returnsDeleteCommand() {
+        assertParseSuccess(parser, "1", new DeleteCommand(INDEX_FIRST_PERSON));
+    }
+
     @Test
     public void parse_invalidArgs_throwsParseException() {
```

Comment:
This test case is wrong. "a", in this case, could be the name of a person.

Also, can there be more tests for invalid arguments? Some scenarios:

* empty string
* negative index
* only spaces

Reply:
in this case, even if "a" is a name of an existing person, it would flag an error because there's no name prefix given `/n`! I have also added more test cases for invalid arguments :)